Source/Plugins/CoherenceViewer/CumulativeTFR.h

```diff
@@ -91,6 +91,11 @@ class CumulativeTFR
 
 	//Array<AudioBuffer<double>> spectrumBuffer;
 
+    // I'm feeling like it might make more sense to have just one of these for power of
+    // all the channels of interest. Also, maybe allow selecting arbitrary pairs of channels
+    // to calculate coherence b/w rather than dividing selected channels into group 1 and
+    // group 2. Thoughts?
+
```

Comment:
Depends. If we want to select 2 groups and do coherence between all combinations I think we leave it as is. If we are selecting specific channel combinations to compare then we change it. Might be a question I ask Alik, what would be most used. I guess in the same sense changing it makes it more robust.

Reply:
OK, I think the simplicity of the interface vs. versatility is definitely an important thing to think about. Having 2 groups the way it is now is probably simpler, but I have some ideas about how we could implement it either way - let's discuss at some point.

On the other hand, I don't thing putting all the power spectra in one variable would increase the complexity of the code, and might actually simplify it (although I haven't tried).

Reply:
Thought about it and I think I like doing a single variable more actually. Then TFR class doesn't care about groups and keeps complexity on that side simpler. Then just have the thread query through combinations.